RELEASE-NOTES.md

```diff
@@ -24,9 +24,50 @@ replication. The rest of the changes are fixes (riak-core claim,
 repl), new features (gsets, participate in coverage, node-confirms),
 and fixes to tests and the build/development process.
 
-### Know Issues
+### Known Issues
 
-@TODO
+#### Advanced.config changes
+
+With the inclusion of Multi-Datacentre Replication in riak-2.2.5 there
+are additional `advanced.config` parameters. If you have an existing
+`advanced.config` you must merge it with the new one from the install
+of riak-2.2.5. Some package installs will simply replace the old with
+new (e.g. .deb), others may leave the old file unchanged. YOU MUST
+make sure that the `advanced.config` contains valid `riak_repl`
+entries.
+
+Example default entries to add to your existing advanced.config:
+
+```
+{riak_core,
+  [
+   {cluster_mgr, {"0.0.0.0", 9080 } }
+  ]},
+ {riak_repl,
+  [
+   {data_root, "/var/lib/riak/riak_repl/"},
+   {max_fssource_cluster, 5},
+   {max_fssource_node, 1},
+   {max_fssink_node, 1},
+   {fullsync_on_connect, true},
+   {fullsync_interval, 30},
+   {rtq_max_bytes, 104857600},
+   {proxy_get, disabled},
+   {rt_heartbeat_interval, 15},
+   {rt_heartbeat_timeout, 15},
+   {fullsync_use_background_manager, true}
+  ]},
+```
+
+Read more about configuring
+[MDC](http://docs.basho.com/riak/kv/2.2.3/configuring/v3-multi-datacenter/)
+replication.
+
+More details about the issue can be found
```

Comment:
@russelldb - you've already provided the useful config directly, no point in directing users to the comment again, also let's make it consistent with the format you've used for others.

```
More details about the issue can be found in riak_repl/782 [2.2.5 - [enoent] - riak_repl couldn't create log dir "data/riak_repl/logs" ](https://github.com/basho/riak/issues/940)
```